AudioContextInterruptedState/explainer.md

```diff
@@ -0,0 +1,212 @@
+# AudioContext Interrupted State
+
+## Authors:
+
+- [Gabriel Santana Brito](https://github.com/gabrielsanbrito)
+- [Steve Becker](https://github.com/SteveBeckerMSFT)
+
+## Participate
+- https://github.com/WebAudio/web-audio-api/issues/2392
+
+## Introduction
+
+The [Web Audio API](https://webaudio.github.io/web-audio-api/) is widely used to add advanced audio capabilities to web applications, like web-based games and music applications. One of the API's features is the [AudioContext interface](https://webaudio.github.io/web-audio-api/#AudioContext), which represents an audio graph. An `AudioContext` can find itself in either of three states: [`"suspended"`](https://webaudio.github.io/web-audio-api/#dom-audiocontextstate-suspended), [`"running"`](https://webaudio.github.io/web-audio-api/#dom-audiocontextstate-running), or [`"closed"`](https://webaudio.github.io/web-audio-api/#dom-audiocontextstate-closed). Once an `AudioContext` is on the `"running"` state, it can only pause media playback by transitioning to the `"suspended"` when, and only when, user code calls [`suspend()`](https://webaudio.github.io/web-audio-api/#dom-audiocontext-suspend) on this `AudioContext`. However, there are situations where we might want to let the User Agent (UA) decide when to interrupt playback - e.g., the proposed [`"media-playback-while-not-visible"`](https://github.com/MicrosoftEdge/MSEdgeExplainers/blob/main/IframeMediaPause/iframe_media_pausing.md) permission policy and the proposed [Audio Session API](https://w3c.github.io/audio-session/); or during a phone call, where the calling application will need exclusive access to the audio hardware. To support these scenarios, we propose adding a new `"interrupted"` state to the [`AudioContextState`](https://webaudio.github.io/web-audio-api/#enumdef-audiocontextstate) enum.

```

Comment:
nits:

"either of three states" --> "one of three states"

"Once an AudioContext is on the "running" state" --> "Once an AudioContext is in the "running" state"

"transitioning to the "suspended" when" -->  "transitioning to the "suspended" state when"